webapp/portlet/src/main/webapp/vue-apps/attach-image/components/form/ImageInputItem.vue

```diff
@@ -64,7 +64,7 @@
         class="ml-0" 
         fab 
         x-small
-        @click="deleteFile">
+        @click="deleteFile($event)">
```

Comment:
This should be sufficient :

```suggestion
        @click.prevent.stop="deleteFile">
```